org.code.toboggan.filesystem/src/org/code/toboggan/filesystem/extensions/file/FSFileMove.java

```diff
@@ -0,0 +1,20 @@
+package org.code.toboggan.filesystem.extensions.file;
+
+import java.nio.file.Path;
+
+import org.code.toboggan.core.api.APIFactory;
+import org.code.toboggan.network.request.extensionpoints.file.IFileMoveResponse;
+
+public class FSFileMove implements IFileMoveResponse {
+
+	@Override
+	public void fileMoved(long fileID, Path newWorkspaceRelativePath) {
```

Comment:
will this parameter #2 actually be a workspace relative path? or will it be absolute? If it will be absolute, please rename the variable. Same with the function below.